pkgs/by-name/cl/clevis/package.nix

```diff
@@ -81,11 +81,16 @@ stdenv.mkDerivation (finalAttrs: {
     clevisZfsFallbackSystemdStage1 = nixosTests.installer-systemd-stage-1.clevisZfsFallback;
   };
 
-  meta = with lib; {
-    description = "Automated Encryption Framework";
+  meta = {
     homepage = "https://github.com/latchset/clevis";
-    changelog = "https://github.com/latchset/clevis/releases/tag/v${version}";
-    license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ AndersonTorres ];
+    description = "Automated Encryption Framework";
+    longDescription = ''
+      Clevis is a pluggable framework for automated decryption. It can be used
+      to provide automated decryption of data or even automated unlocking of
+      LUKS volumes.
+    '';
+    changelog = "https://github.com/latchset/clevis/releases/tag/v${finalAttrs.version}";
```

Comment:
Nit: This `finalAttrs` change should be part of the `clevis: rework - finalAttrs` commit.

Reply:
OK